packages/tabs/src/Tabs/Tabs.types.ts

```diff
@@ -64,16 +64,16 @@ export interface TabsProps
   inlineChildren?: React.ReactNode;
 
   /**
-   * Index of the Tab that should appear active. If value passed to selected prop, component will be controlled by consumer.
+   * Index or name of the Tab that should appear active. If value passed to selected prop, component will be controlled by consumer.
    */
-  selected?: number;
+  selected?: number | string;
 
   /**
-   * Callback to be executed when Tab is selected. Receives index of activated Tab as the first argument.
+   * Callback to be executed when Tab is selected. Receives index or name of activated Tab as the first argument.
    *
    * @type (index: number) => void
    */
-  setSelected?: React.Dispatch<number>;
+  setSelected?: React.Dispatch<number> | React.Dispatch<string>;
```

Comment:
Can we update these with a type param. so that they're always in sync (i.e. `selected: T` and `setSelected: Dispatch<T>`